ntc_templates/templates/zyxel_os_cfg_static_route_get.textfsm

```diff
@@ -0,0 +1,19 @@
+Value INDEX (\d+)
+Value NAME (.+?)
+Value ENABLED (0|1)
+Value IPVER (IPv4|IPv6)
+Value DESTINATION ((([0-9]{1,3}\.){3}[0-9]{1,3}(/\d{1-2})?)?)
```

Comment:
Similar to others, if this doesn't exist in some cases, need to have tests and likely an alternative strategy.